lsp-ui-imenu.el

```diff
@@ -112,62 +149,120 @@
         (when (overlayp lsp-ui-imenu-ov)
           (delete-overlay lsp-ui-imenu-ov))))))
 
+(defun lsp-ui-imenu--move-to-line-beginning ()
+  (-when-let* ((padding (get-char-property (point) 'padding))
+               (bars-count (get-char-property (point) 'bars-count)))
```

Comment:
I don't fully understand the code, but it seems to me `depth` is a better name than `bars-count` (which counts confusing "invisible ones")

Reply:
depth is a good name. Done.